homeassistant/components/device_tracker/__init__.py

```diff
@@ -398,15 +398,32 @@ def update(self):
 
 def load_config(path: str, hass: HomeAssistantType, consider_home: timedelta):
     """Load devices from YAML configuration file."""
+    dev_schema = vol.Schema({
```

Comment:
Include the whole file in 1 schema that will also convert the names to lowercase